libs/feat/design-system/components/atoms/Fields/Checkbox/index.d.ts

```
@@ -0,0 +1,7 @@
+import { Checkbox as UiCheckbox } from '@lssm/ui-kit/ui/checkbox';
+import type { ComponentProps } from 'react';
+interface CheckboxProps extends ComponentProps<typeof UiCheckbox> {}
+export declare const Checkbox: (
+  props: CheckboxProps,
+) => import('react/jsx-runtime').JSX.Element;
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)